sccp/figures/common.py

```diff
@@ -485,6 +487,19 @@ def openPf2(rank: int, dataName: str, optProjs = False):
         
     return weight, factors, projs
 
+def saveUMAP(fit_points, rank:int, dataName: str):
+    """Saves UMAP points locally, large files uploaded manually to opt"""
+    f_name = join(path_here, "data/"+dataName+"/"+dataName+"_UMAPCmp"+str(rank)+".sav")
+    pickle.dump(fit_points, open(f_name, 'wb'))
+
+def openUMAP(rank: int, dataName: str, opt = True):
```

Comment:
it *might* make more sense to have the opt default set to false here; but rn I am the only one using this function so this is the easiest

Reply:
@pascoesean Yeah making it True will probably make more sense in the end